### ISSUE TITLE
I got problem with cloned values from base class.

### ISSUE THREAD

**Issue by IgorMamushin** [marcelltoth/ObjectCloner#2]
When i try cloned object with base class, it don't copy values from base class. For example List, Array, other classes and etc.

**Comment by IgorMamushin** [comment]
![122222](https://user-images.githubusercontent.com/11444813/70317302-ed7e4f00-1836-11ea-808f-684b39c28f2d.jpg)

```
public class SecondClass
{
     public int Value { get; set; }
}

public class BaseClass
{
     public IList<SecondClass> ListClassesBase { get; set; }
}
         
private class MainTestClass : BaseClass
{
      public IList<SecondClass> ListClasses { get; set; }
}
```

**Comment by marcelltoth** [comment]
Hi Igor,

Thank you for submitting the issue. I will look into it.

**Comment by marcelltoth** [comment]
Fixed in 2.2.2, thank you again for your cooperation.

If you spot any other bug or possible enhancement, be sure to create new issues / PRs

**Comment by IgorMamushin** [comment]
Thanks a lot, it work well.
